spec/unit/puppet/provider/mysql_database/mysql_spec.rb

```diff
@@ -72,7 +72,7 @@
 
   describe 'exists?' do
     it 'checks if database exists' do
-      expect(instance.exists?).to be_truthy
+      expect(instance.exists?).to be_truthy # rubocop:disable RSpec/PredicateMatcher
```

Comment:
can this not be fixed ?

Reply:
Maybe something like:
`expect(instance).to be_exists` ?